src/Block/Adminhtml/Project/Tab/Categories.php

```diff
@@ -100,6 +108,10 @@ protected function _prepareCollection(): Grid
             $categoryCollection->addFieldToFilter('entity_id', ['in' => $categoryIds]);
         }
         $categoryCollection->setStoreId($this->projectGetter->getProject()->getSourceStoreId());
+        $this->eventManager->dispatch(
+            'easytranslate_prepare_categories_collection',
```

Comment:
```suggestion
            'easytranslate_prepare_category_collection',
```

To match Magento event naming.